spark/src/test/scala/org/apache/spark/sql/delta/skipping/clustering/ClusteredTableDDLSuite.scala

```diff
@@ -825,6 +826,45 @@ trait ClusteredTableDDLSuiteBase
     }
   }
 
+  test("validate RESTORE on clustered table") {
```

Comment:
Is there a column mapping enabled variant for these tests? If not let's add one since column names are maninulated

Reply:
Sure, added them to column mapping's selected tests.